drivers/modem/modem_cellular.c

```diff
@@ -2275,6 +2355,8 @@ MODEM_CHAT_SCRIPT_DEFINE(sqn_gm02s_periodic_chat_script,
 		.init_chat_script = &swir_hl7800_init_chat_script,                                 \
 		.dial_chat_script = &swir_hl7800_dial_chat_script,                                 \
 		.periodic_chat_script = &swir_hl7800_periodic_chat_script,                         \
+		.shell_pipelink = MODEM_PIPELINK_DT_GET(DT_DRV_INST(inst), shell),                 \
+		.gnss_pipelink = MODEM_PIPELINK_DT_GET(DT_DRV_INST(inst), gnss),                   \
```

Comment:
Not introduced by this PR, but I'm thinking it could be nice to put back the parts common to all supported modems in a single place instead of having to repeat them for every modem.

Reply:
Agree, my plan is to refactor this driver to be split into smaller modules, connected by modem pipelinks, which should remove a bunch of duplicate code from this driver :) but, in a future PR